app/assets/stylesheets/_loading_indicator.sass

```diff
@@ -0,0 +1,136 @@
+$half-inner-balance-height: 0.312rem
```

Comment:
Het is me niet duidelijk wat dit betekent. Kun je een duidelijkere naam verzinnen? Juist omdat het zo'n constante is lijkt het me wel fijn om te weten hoe je tot 0.312 bent gekomen